modules/indexing/src/main/java/org/apache/ignite/internal/visor/verify/ValidateIndexesClosure.java

```diff
@@ -482,28 +476,28 @@ private IndexIntegrityCheckIssue integrityCheckIndexPartition(CacheGroupContext
     }
 
     /**
-     * @param grpCtx Group context.
-     * @param part Local partition.
+     * @param grpCtxWithPart Group context and partition id.
      */
     private Future<Map<PartitionKey, ValidateIndexesPartitionResult>> processPartitionAsync(
-        final CacheGroupContext grpCtx,
-        final GridDhtLocalPartition part
+        final T2<CacheGroupContext, GridDhtLocalPartition> grpCtxWithPart
     ) {
         return calcExecutor.submit(new Callable<Map<PartitionKey, ValidateIndexesPartitionResult>>() {
-            @Override public Map<PartitionKey, ValidateIndexesPartitionResult> call() throws Exception {
-                return processPartition(grpCtx, part);
+            @Override public Map<PartitionKey, ValidateIndexesPartitionResult> call() {
+                return processPartition(grpCtxWithPart);
             }
         });
     }
 
     /**
-     * @param grpCtx Group context.
-     * @param part Local partition.
+     * @param grpCtxWithPart Group context and partition id.
      */
     private Map<PartitionKey, ValidateIndexesPartitionResult> processPartition(
-        CacheGroupContext grpCtx,
-        GridDhtLocalPartition part
+        final T2<CacheGroupContext, GridDhtLocalPartition> grpCtxWithPart
```

Comment:
Wasn't able to understand the purpose of the change. We have one more object as a result.